docker-hub/billing/pricing-pull-rate-limit-faqs.md

```diff
@@ -0,0 +1,559 @@
+
+### What are the differences between the Team and Free Team plans?
+
+For details on the differences for each plan option, please see [https://docs.docker.com/docker-hub/billing/](https://docs.docker.com/docker-hub/billing/).
+
+### Do collaborator limits differ between the Free and Pro plans?
+
+The Free plan includes unlimited collaborators for public repositories and 0 collaborators for private repositories. Pro includes unlimited collaborators for public repositories and 1 unique collaborator for private repositories. Note that a collaborator on a private repository within a Pro plan can only be used with a service account.
+
+For details on the differences between plans, see [https://www.docker.com/pricing](https://www.docker.com/pricing).
+
+For more information on service accounts visit our [docs](/docker-hub/repos/#service-accounts) page.
+
+### How can I create a new Docker Hub account?
+
+You can create a new account at [https://www.docker.com/pricing](https://www.docker.com/pricing) where you can choose a plan for Individuals or a plan for Teams.
+
+### How do I upgrade to a Pro plan from a legacy individual plan?
+
+Upgrading your legacy plan to a Pro plan offers you unlimited public repos, unlimited private repos, and unlimited collaborators. For information on how to upgrade to a Pro plan from a legacy plan, see [Upgrade to a Pro plan](/docker-hub/billing/upgrade/#upgrade-to-a-pro-plan).
+
+### How do I upgrade to a Team plan from a legacy organization plan?
+
+Upgrading your legacy plan to a Team plan offers you unlimited private repos, unlimited teams, and 3 parallel builds. For information on how to upgrade to a Team plan from a legacy (per-repository) plan, see [Upgrade to a Team plan](/docker-hub/billing/upgrade/#upgrade-to-a-team-plan).
+
+### How do downgrades from a Pro or Team plan work?
+
+When you downgrade your Pro or Team plan, changes are applied at the end of your billing cycle. For example, if you are currently on a Team plan which is billed on the 8th of every month and you choose to downgrade to a Free Team plan on the 15th, your Team plan will be active until 7th of the following month. You will be transferred to a Free Team plan on the 8th of the following month. Please refer to the next two questions below for instructions on how to downgrade your account.
+
+### How do I downgrade from a Team plan to a Free Team plan?
+
+Before you downgrade to a Free plan, ensure that your organization details are updated to reflect features available in the Free plan. For example, you may need to reduce the number of team members and convert any private repositories to public repositories. For information on what’s included in the Free plan, see [https://docs.docker.com/docker-hub/billing/](/docker-hub/billing/).
```

Comment:
Should we link this to the www pricing page instead of docs for details on what's included in the Free plan?